### PR TITLE
{lib}[GCCcore/8.2.0] libsodium v1.0.17

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)
Minor update of libsodium to v1.0.17

### CODE REVIEW

File: easybuild/easyconfigs/l/libsodium/libsodium-1.0.17-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsodium'
+version = '1.0.17'
+
+homepage = 'http://doc.libsodium.org/'
```
Comment (Micket) [inline easybuild/easyconfigs/l/libsodium/libsodium-1.0.17-GCCcore-8.2.0.eb]: switch to https?

Comment (lexming) [reply easybuild/easyconfigs/l/libsodium/libsodium-1.0.17-GCCcore-8.2.0.eb]: Done :+1:

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/5322594d7487156a2956b5b9eb6cc8bf for a full test report.

**Comment by Micket** [comment]
Going in, thanks @lexming!

**Pull request merged**